community/README.md

```diff
@@ -13,7 +13,7 @@ documentation for more details.
 Information about SIGs is sourced from the [sigs.yaml](./sigs.yaml) file.
 
 To generate all the SIG documentation and update the OWNERS files from the
-source file, use the `generate` or `generate-dockerized` targets in the
+source file, use the `generate` target in the
```

Comment:
Note that the command below is just `make` (i.e. using the `default` target, that currently points to `generate`). This might be a bit counfusing.

```suggestion
source file, use the `generate` target (which is the default target) in the
```

or mention the target name explicitly below.